Analysis/include/MvaVariables.h

```diff
@@ -0,0 +1,135 @@
+        VAR("deta_hbbmet", bb.eta()-event.pfMET_p4.eta());
+        VAR("abs_deta_hbbhtautau", std::abs(bb.eta()-event.SVfit_p4.eta()));
+        VAR("deta_hbbhtautau", bb.eta()-event.SVfit_p4.eta());
+        VAR("dR_l1l2", ROOT::Math::VectorUtil::DeltaR(event.p4_1, event.p4_2));
+        VAR("dR_b1b2", ROOT::Math::VectorUtil::DeltaR(event.jets_p4[0], event.jets_p4[1]));
+        VAR("dR_l1met", ROOT::Math::VectorUtil::DeltaR(event.p4_1, event.pfMET_p4));
+        VAR("dR_l2met", ROOT::Math::VectorUtil::DeltaR(event.p4_2, event.pfMET_p4));
+        VAR("dR_l1l2met", ROOT::Math::VectorUtil::DeltaR(leptons, event.pfMET_p4));
+        VAR("dR_htautaumet", ROOT::Math::VectorUtil::DeltaR(event.SVfit_p4, event.pfMET_p4));
+        VAR("dR_hbbmet", ROOT::Math::VectorUtil::DeltaR(bb, event.pfMET_p4));
+        VAR("dR_hbbhtautau", ROOT::Math::VectorUtil::DeltaR(bb, event.SVfit_p4));
+        VAR("dR_b1b2Pt_hbb", (ROOT::Math::VectorUtil::DeltaR(event.jets_p4[0], event.jets_p4[1]))*bb.Pt());
+        VAR("dR_l1l2Pt_htautau", ROOT::Math::VectorUtil::DeltaR(event.p4_1, event.p4_2)*event.SVfit_p4.Pt());
+        VAR("mass_l1l2met", ROOT::Math::VectorUtil::InvariantMass(leptons,event.pfMET_p4));
+        VAR("mass_htautau", event.SVfit_p4.M());
+        VAR("mass_l1l2", (event.p4_1+event.p4_2).M());
+        VAR("mass_hbb", bb.M());
+        VAR("MT_l1", Calculate_MT(event.p4_1,event.pfMET_p4));
+        VAR("MT_l2", Calculate_MT(event.p4_2,event.pfMET_p4));
+        VAR("MT_htautau", Calculate_MT(event.SVfit_p4, event.pfMET_p4));
+        VAR("MT_l1l2", Calculate_MT(leptons, event.pfMET_p4));
+        VAR("MT_tot", Calculate_TotalMT(event.p4_1,event.p4_2,event.pfMET_p4)); //Total transverse mass
+        VAR("MT2", std::min(Calculate_MT2(event.p4_1,event.p4_2,event.jets_p4[0], event.jets_p4[1], event.pfMET_p4),Calculate_MT2(event.p4_1, event.jets_p4[1], event.p4_2,event.jets_p4[0], event.pfMET_p4))); //Stransverse mass
+//        VAR("mass_H", ROOT::Math::VectorUtil::InvariantMass(bb,event.SVfit_p4));
+        VAR("mass_top1", four_bodies::Calculate_topPairMasses(event.p4_1, event.p4_2, event.jets_p4[0], event.jets_p4[1], event.pfMET_p4).first);
+        VAR("mass_top2", four_bodies::Calculate_topPairMasses(event.p4_1, event.p4_2, event.jets_p4[0], event.jets_p4[1], event.pfMET_p4).second);
+//        VAR("MX", four_bodies::Calculate_MX(event.p4_1, event.p4_2, event.jets_p4[0], event.jets_p4[1], event.pfMET_p4));
+        VAR("dR_l1l2_boosted", four_bodies::Calculate_dR_boosted(event.p4_1, event.p4_2, event.SVfit_p4));
+        VAR("dR_b1b2_boosted", four_bodies::Calculate_dR_boosted(event.jets_p4[0], event.jets_p4[1], bb));
+        VAR("phi", four_bodies::Calculate_phi(event.p4_1,event.p4_2,event.jets_p4[0], event.jets_p4[1], event.SVfit_p4, bb));
+        VAR("costheta_star_leptons", four_bodies::Calculate_cosThetaStar(event.SVfit_p4, bb));
+        VAR("costheta_star_bjets", four_bodies::Calculate_cosThetaStar(bb, event.SVfit_p4));
```

Comment:
costheta_star(h1) = -costheta_star(h2) by construction, so in this case combinatorics not needed.